### PR TITLE
Consumption Reports

### PR DESCRIPTION
**Pull request by benjaminach**

### CODE REVIEW

File: src/_posts/platform/internals/2000-01-01-consumption-reports.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+The amounts shown are frozen at the end of each billing period. An application transferred during the month is billed to its new owner, even if you can see its consumption for the period prior to its transfer.
+{% endnote %}
+
+## Current or past periods
+
+This report shows the consumption of all your applications for the current billing period and is updated daily at midnight UTC.
+
+When a billing period is over and the invoice is issued, you can explore your consumption to identify the main cost drivers.
+
+Please note that the consumption report document **is not an invoice**, but a breakdown of your consumption **excluding VAT**. It does not take into account the free trial period or any vouchers or credits.
```
Comment (Anouchka-M) [inline src/_posts/platform/internals/2000-01-01-consumption-reports.md:24]:
```suggestion
Please note that the consumption report document **is not an invoice**, but a breakdown of your consumption **excluding VAT**. It takes neither the free trial period nor any vouchers or credits into account .
```

File: src/_posts/platform/internals/2000-01-01-consumption-reports.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Group consumption by app or ressource
+
+Consumption are grouped by **App** by default. This makes it easy to identify the applications that consume the most resources and represent the largest cost centre.
```
Comment (Anouchka-M) [inline src/_posts/platform/internals/2000-01-01-consumption-reports.md:28]:
```suggestion
Consumption are grouped by **App** by default. This makes it easy to identify which applications consume the most resources and represent the largest cost centre.
```

**Comment by Anouchka-M** [comment]
Two non blocking suggestions of a wording which seems a bit "smoother" to me.

File: src/_posts/platform/internals/2000-01-01-consumption-reports.md (hunk 1 above)
Comment (benjaminach) [reply src/_posts/platform/internals/2000-01-01-consumption-reports.md:24]: @Anouchka-M I have used the same translation as the one in the dashboard. Could you also make the change in the dashboard?

**Review by Anouchka-M — APPROVED** [review]

**Review by ksol — APPROVED** [review]

**Pull request closed without merging**